azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/CallComposite.java

```diff
@@ -335,22 +338,33 @@ private void launchComposite(final Context context,
         UUID groupId = null;
         String meetingLink = null;
         final CallType callType;
+        List<CommunicationIdentifier> participants = null;
 
         final CallCompositeJoinLocator locator = remoteOptions.getLocator();
-        if (locator instanceof CallCompositeGroupCallLocator) {
-            callType = CallType.GROUP_CALL;
-            groupId = ((CallCompositeGroupCallLocator) locator).getGroupId();
-        } else {
-            callType = CallType.TEAMS_MEETING;
-            meetingLink = ((CallCompositeTeamsMeetingLinkLocator) locator).getMeetingLink();
+        if (locator != null) {
+            if (locator instanceof CallCompositeGroupCallLocator) {
+                callType = CallType.GROUP_CALL;
+                groupId = ((CallCompositeGroupCallLocator) locator).getGroupId();
+            } else {
+                callType = CallType.TEAMS_MEETING;
+                meetingLink = ((CallCompositeTeamsMeetingLinkLocator) locator).getMeetingLink();
+            }
+        }
+        else {
+            callType = CallType.ONE_ON_ONE_CALL;
+            participants = new ArrayList<>();
+            for (String participantId : remoteOptions.getStartCallOptions().getParticipants()) {
+                participants.add(new CommunicationUserIdentifier(participantId));
```

Comment:
Please change to CommunicationIdentifier.fromRawId(participantId)

Reply:
We do not need to convert here. We can use list of string until reaches to service layer